public/controllers/login-signup.js

```diff
@@ -85,6 +86,31 @@ controller.controller('Login', ['ConfigService', '$scope', '$http', '$location',
                 alert("login failed, reason: " + err.reason);
             });
         };
+        $scope.resetPassword = function(){
+            var url = ConfigService.serverIp + '/forgetPassword';
+            $http.post(url, {username: $scope.forgottenUsername}).success(function(response) {
+                alert(response);
+            }).error(function (err) {
+                alert("Some Error occured: " + err.reason);
+            });
+        };
+
+        $scope.updateNewPassword= function(){
+            if($scope.pass1!= null && $scope.pass2!= null && $scope.pass1===$scope.pass2){
+                var url = ConfigService.serverIp + '/resetPassword';
+                $scope.userId = $routeParams.id;
+                $http.post(url, {pass: $scope.pass1,userId: $scope.userId}).success(function(response) {
+                    console.log(response);
```

Comment:
user ko bhi tou bata na k password reset ho gaya hai 😛  and also route him to the login page after that. password rest honay k baad iss page par khara rehne ka kya faida ?:p